content/docs/ops/runbook/restoring-rds.md

```diff
@@ -0,0 +1,80 @@
+
+Once the tenant consents, identify their Organization and Space.  Then perform the following:
+```sh
+cf target -s SPACE -o ORGANIZATION
+```
+
+Print environment variables for the affected application:
+```sh
+cf env APP
+```
+
+The resulting JSON contains the "host" name within the credentials map.  This will be the database identifier needed below for restoration.
+
+### Platform Applications
+Databases created by Terraform store their credentials in a Terraform State Bucket.  This can be found in S3:
+```sh
+aws s3 cp s3://${TF_STATE_BUCKET}/cf-${ENVIRONMENT}/terraform.tfstate tmp/state.file
+```
+
+To retrieve the RDS instance provisioned for `staging concourse`:
+```sh
+cat tmp/sample.file | grep staging_concourse_rds_host | less
+```
+
+The value displayed is the database to restore.  To restore the BOSH instnace grep for `bosh_rds_host_curr`.
+
+NOTE: These values can also be found in the AWS RDS console.  This can be faster.
+
+## Restore the database
+Refer to the [RDS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RestoreFromSnapshot.html) for restoring a database.
+
+For the step that specifies choosing the database to restore, this database will be the respective database we found in the steps above.
+
+### CloudFoundry Applications
+The database restore process will create a new instance with the same credentials.  Create a user provided service with the credentials:
+```sh
+cf create-user-provided-service database-restore -p '{"db_name": "DATABASE_NAME", "host": "DATABASE_HOST", "password":"DATABASE_PASSWORD", "port": "5432", "uri": "postgres://DATABASE_USERNAME:DATABASE_PASSWORD@DATABASE_HOST:5432/DATABASE_NAME", "username": "DATABASE_USERNAME"}'
+```
+
+The methods described at [Relational Databases](https://cloud.gov/docs/services/relational-database/) can be used to connect to the restoration copy and restore data as needed.
+
+### Bosh + Concourse Database Restore Procedure
+For these deployments we must performa a Terraform import to restore the database.  First follow the steps above to restore the database in question.
+
+Now run `terraform init` and `terraform import` in the directory for the respective Terraform module.  For development Bosh, eg run these commands:
+
+```sh
+cd terraform/stacks/main
+terraform init -backend=true -backend-config=encrypt=true -backend-config=bucket=terraform-state -backend-config=key=development/terraform.tfstate
+terraform state rm module.stack.module.base.module.rds.aws_db_instance.rds_database
+terraform import module.stack.module.base.module.rds.aws_db_instance.rds_database my-restored-db-id
+```
+
+where my-restored-db-id is the database instance defined during the restore procedure.
+
+Now run the plan job pipeline in concourse for the respective database. If Concourse is not available (due to the DB being unavailable) you will need to run `terraform plan` from the command line.
```

Comment:
Concourse might be unavailable for other reasons. WDYT about dropping the parenthetical here?

Reply:
I think good things, myself.